algosdk/v2client/models/simulate_request.py

```diff
@@ -29,11 +62,13 @@ def __init__(
         allow_more_logs: bool = False,
         allow_empty_signatures: bool = False,
         extra_opcode_budget: int = 0,
+        exec_trace_config: SimulateTraceConfig = SimulateTraceConfig(),
```

Comment:
minor: I'd prefer if this defaults to None, then in the function body you call the `SimulateTraceConfig` constructor

Reply:
see [pr comment response](https://github.com/algorand/py-algorand-sdk/pull/505/commits/9c9baade970c1cdae430a231df7cdd8279464fb9)